packages/components/src/base-control/index.styles.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+
+export const Control = styled.div`
+	& + & {
+		margin-bottom: 16px;
+	};
+`;
+
+export const Field = styled.div`
+	margin-bottom: 8px;
+`;
+
+export const Label = styled.label`
+	/* text({variant: 'body.small'}) */
```

Comment:
Waiting on https://github.com/WordPress/gutenberg/pull/18495 which implements [a-consistent-type-scale-for-wordpress](https://make.wordpress.org/design/2019/10/11/proposal-a-consistent-type-scale-for-wordpress/)